src/contexts/AccountContextProvider.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+
+const AccountContext = React.createContext({
+  account: {
+    id: null,
+    name: null,
+    avatar: null,
+  },
+  favoriteVideos: [],
+  addFaveVideo: () => {},
+  accountChange: () => {},
+});
```

Comment:
We can get account information from localStorage by `store.get('account')` as initial value instead of reset its value whenever we refresh the page